af/shovel/centrifugation.py

```diff
@@ -1685,6 +1685,44 @@ def update_explorer_metrics(pgconn, bucket):
 UPDATE
 SET count = EXCLUDED.count;''', [bucket])
 
+        # See:  016-ooexpl_wc_confirmed.install.sql
+        update_ooexpl_wc_confirmed = '''
+INSERT INTO ooexpl_wc_confirmed
+SELECT
+COALESCE(SUM(CASE WHEN confirmed = TRUE THEN 1 ELSE 0 END), 0) as confirmed_count,
+COUNT(*) as msm_count,
+date_trunc('day', test_start_time) as test_day,
+probe_cc,
+probe_asn
+FROM (
+	SELECT
+	DISTINCT input as input,
+	test_start_time,
+	probe_cc,
+	probe_asn,
+	bool_or(confirmed) as confirmed
+	FROM measurement
+	JOIN input ON input.input_no = measurement.input_no
+	JOIN report ON report.report_no = measurement.report_no
+	WHERE test_start_time >= current_date - interval '2 day'
+	AND test_start_time < current_date - interval '1 day'
```

Comment:
It is not necessary that the `current_date` is going to be the time at which this is executed. It's better to use the `bucket_date` constraint to INSERT.